### PR TITLE
Update configuration documentation

### PR DESCRIPTION
**Pull request by jhftrifork**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,109 @@
...
+After satisfying the above requirements,
+start the Logstash framework by starting a Docker container from your `mesos/logstash-scheduler` image,
+like this:
+
+```bash
+> docker run mesos/logstash-scheduler \
+    --zk-url=zk://123.0.0.12:5181/logstash \
+    --zk-timeout=20000 \
+    --framework-name=logstash \
+    --webserver-port=10000 \
```
Comment (mwl) [inline README.md]: Has been deleted

File: README.md [hunk 2]
```diff
@@ -1,51 +1,109 @@
...
+    --user=root \
+    --logstash.heap-size=64 \
+    --logstash.elasticsearch-url=elasticsearch.service.consul:1234 \
+    --executor.cpus=0.5 \
+    --executor.heap-size=128 \
+    --enable.failover=false \
+    --enable.collectd=true \
+    --enable.syslog=true \
+    --enable.file=true \
+    --file.path='/var/log/*,/home/jhf/example.log'
```
Comment (mwl) [inline README.md]: It's `--executor.file-path=…`

File: README.md [hunk 3]
```diff
@@ -1,51 +1,109 @@
...
+| `--executor.heap-size=H`         | `EXECUTOR_HEAP_SIZE=H`         | `64`       | The memory allocation pool for the Logstash executor will be limited to `H` megabytes                                      |
+| `--enable.failover=F`            | `ENABLE_FAILOVER=F`            | `true`     | If `F` is `true`, all executors and tasks will remain running after this scheduler exits FIXME what's the format for `F`   |
+| `--enable.collectd=C`            | `ENABLE_COLLECTD=C`            | `false`    | If `C` is `true`, Logstash will listen for collectd events on TCP/UDP port 5000 on all executors                           |
+| `--enable.syslog=S`              | `ENABLE_SYSLOG=S`              | `false`    | If `S` is `true`, Logstash will listen for syslog events on TCP port 514 on all executors                                  |
+| `--enable.file=F`                | `ENABLE_FILE=F`                | `false`    | If `F` is `true`, each line in files matching the `--file.path` pattern will be treated as a log event                     |
```
Comment (mwl) [inline README.md]: Shouldn't it be `=B` for boolean?

File: README.md (hunk 1 above)
Comment (jhftrifork) [inline README.md]: Correct -- removed it

File: README.md (hunk 2 above)
Comment (jhftrifork) [inline README.md]: My bad; fixed it

File: README.md (hunk 3 above)
Comment (jhftrifork) [inline README.md]: Could be. I changed all the boolean values to B

File: README.md [hunk 4]
```diff
@@ -53,48 +111,50 @@ Since the executor runs inside its own docker container it will try to reach its
...
-     }
-   },
-   "env": {
-     "JAVA_OPTS": "-Dmesos.logstash.framework.name=logstash_framework -Dmesos.zk=zk://<zkserver:port>,<zkserver:port>/mesos"
-   }
- }
- ```
- 
- Note: See <a href="#fw_configuration">Framework options</a> for all available options.
+```json
```
Comment (mwl) [inline README.md]: This needs a little wax on-wax off. @sadovnikov Could you post your latest Marathon config here?

**Comment by mwl** [comment]
LGTM

**Pull request merged**